test/check-machines-multi-host-consoles

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/common/pywrap", sys.argv)
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2021 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import testlib
+from machineslib import VirtualMachinesCase
+
+
+def start_machine_troubleshoot(b, new=False, known_host=False, password=None):
```

Comment:
This is copied from cockpit's check-shell-multi-machine . OK for now, but can you please create a cockpit PR that moves these to testlib.py? Especially as it changed recently, and may again at some point.

Reply:
https://github.com/cockpit-project/cockpit/pull/19659 . Need to retry some tests, but I expect to land this in 30 mins

Reply:
Landed. I triggered a lib update to get this out of the way: PR #1329 . Once that is in, you can rebase and remove the duplication.

Reply:
Lib update landed.